workflow-examples/README.md

```diff
@@ -46,6 +50,11 @@ Once the Jar has been compiled, ensure it's added to the pom.xml of the workflow
 	 <artifactId>workflow-examples</artifactId>
 	 <version>${parodos.version}</version>
 </dependency>
+<dependency>
+    <groupId>com.redhat.parodos</groupId>
```

Comment:
shouldn't this be updated to `dev.parodos` ?

Reply:
@masayag fixed